person.rb

```diff
@@ -2,29 +2,34 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
+
+  has_many :rentals
+  has_many :books, through: rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
-    @id = Random.rand(1..100)
+    @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
```

Comment:
- [ ] Kindly note that a person has many rentals, so it will make sense if  `*@rentals is initialized.` so that all rentals made my a person will be pushed to the rentals array 👍🏽. e.g `@rentals = []`